### DISCUSSION TITLE
Smtp built-in connector - How to return error message

### DISCUSSION THREAD

**Discussion by cveld** [Azure/logicapps#671] [Q&A]
Whenever the smtp built-in connector fails, how can we return the error message to the client?

**Comment by msitsoftware** [comment]
You can achieve it when you set the "Run after" Property of the following shape to "has failed". 

<img width="660" alt="image" src="https://user-images.githubusercontent.com/69357066/200414214-bba360d8-cac5-4070-9dda-452fd3e898ec.png">

In my example I used a simple Request-Response Pattern just to demonstrate. In the Response-Body you can get the error message via @outputs('Send_Email')
 
<img width="791" alt="image" src="https://user-images.githubusercontent.com/69357066/200414115-daf923ba-fdc1-4264-adda-3bfd7fa0e8ed.png">
